staff/almu-itur/js-arrays/fill.js

```diff
@@ -0,0 +1,44 @@
+function fill(){
```

Comment:
OK, but you should try to do it shorter, its possible to do the same using less code. I like how Sean resolved it.  Take a look. : )